### DISCUSSION TITLE
Bot says it is the only member of my server?

### DISCUSSION THREAD

**Discussion by penneShells** [Rapptz/discord.py#5948] [General]
So, im following a tutorial using this library, and it gives an example of how to print out the members of the server. The code looks like this:

```py
import os

import discord
from dotenv import load_dotenv

load_dotenv()
TOKEN = os.getenv("DISCORD_TOKEN")
GUILD = os.getenv("DISCORD_GUILD")

client = discord.Client()

@client.event
async def on_ready():
    for guild in client.guilds:
        if guild.name == GUILD:
            break

    print(
        f"{client.user} is connected to the following guild:\n"
        f"{guild.name}(id: {guild.id})"
    )
    members = "\n - ".join([member.name for member in guild.members])
    print(f"Guild Members:\n - {members}")


    for guild in client.guilds:
        for member in guild.members:
            print(member)
            print(len(guild.members))
client.run(TOKEN)

```
But the output I get is this:

```
The bot is connected to the following guild:
(my guild name)
Guild Members:
Bot
Bot
1
```

Why does it think it is the only member? I am one of three others in the group, why can't it see them?

Thanks

**Comment by penneShells** [comment]
Sorry for the formatting, must have done something wrong, tell me if i should edit it for legibility sake. Sorry!

**Comment by Rapptz** [comment]
For future references, to do a codeblock spanning multiple lines, 3 backticks are on each side. See the [guide](https://guides.github.com/features/mastering-markdown/) for more information.

Getting that out of the way, in order for you to get member information outside of your bot you need to [enable the members intent](https://discordpy.readthedocs.io/en/latest/intents.html#where-d-my-members-go).
